wallet-api/tutorial/Tutorial.md

```diff
@@ -190,81 +209,191 @@ In the `Collect` case, the current slot must be between `deadline` and `collecti
               in
 ```
 
+**Note (Operators in On-Chain Code)** We can use the operators `>`, `<`, `>=`, `<=` and `==` to compare `Int` values in PLC without having to define them in the script itself, as we did with `&&`. The compiler plugin that translates Haskell Core to Plutus Core knows about those operators because `Int` is a primitive type in Plutus Core and operations on it are built in. `Bool` on the other hand is treated like any other user-defined data type, and all functions that operate on it must be defined locally. More details can be found in the [PlutusTx tutorial](https://github.com/input-output-hk/plutus/blob/master/plutus-tx/tutorial/Tutorial.md).
```

Comment:
Maybe we shouldn't actually do this? We could instead expose the builtins via the prelude for consistency...

Reply:
Yes, not having them as operators is a bit annoying, but not necessarily fatal. And it would be simpler. Something to think about, anyway.

Reply:
Yeah you might be right, I think it's the fact that `>` and friends work without imports that has encouraged me to unwrap newtypes like `Slot` and compare the `Int`s directly, which isn't good. 

I'm re-export them from the Prelude.

Reply:
(That can happen in another PR, though)